src/client/src/commons/form/borehole/boreholeForm.js

```diff
@@ -27,6 +27,7 @@ import Hydrotest from "./hydrogeology/hydrotest";
 import { AlertContext } from "../../alert/alertContext";
 import FieldMeasurement from "./hydrogeology/fieldMeasurement";
 import ChronostratigraphyPanel from "./chronostratigraphy/chronostratigraphyPanel";
+import LithostratigraphyPanel from "./chronostratigraphy/lithostratigraphyPanel";
```

Comment:
Q: kann man einen allgemeineren Namen für den Ordner finden, oder sonst 2 machen? Derzeit ist Lithostratigraphie noch unter chronostratigraphy versorgt.

Reply:
In zwei Ordner aufteilen macht keinen sinn, da die meisten Komponenten von chronostratigraphy und lithostratigraphy verwendet werden. Ich benenne den Ordner zu _stratigraphy_ um.